packages/gatsby-theme-store/src/components/ProductPage/useAsyncProduct.ts

```diff
@@ -17,12 +18,17 @@ export const useAsyncProduct = (
   variables: AsyncProductQueryQueryVariables,
   options?: Options
 ) => {
+  const { regionId } = useRegion()
```

Comment:
since regionId is a query variable, shouldn't it come with the `variables` parameter ?